drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c

```diff
@@ -134,6 +159,13 @@ static enum wifi_nrf_status wifi_nrf_fmac_deinit_rx(struct wifi_nrf_fmac_dev_ctx
 
 	fpriv = fmac_dev_ctx->fpriv;
 
+#ifdef CONFIG_NRF700X_RX_WQ_ENABLED
+	wifi_nrf_osal_tasklet_free(fmac_dev_ctx->fpriv->opriv,
+				     fmac_dev_ctx->rx_tasklet);
```

Comment:
Indentation

Reply:
Tried again, aligning to open brace vs. not exceeding 100 lines vs. using tabs as much as possible, its impossible to do this alignment anyother (AFAIK), so, closing this comment as checkpatch is passing.